templates/commands/render.go

```diff
@@ -53,18 +53,34 @@ const (
 
 type Render struct {
 	cli.BaseCommand
+	flags renderFlags
```

Comment:
Do you want this to be a pointer? As there are more-and-more flags, it will get expensive to copy this.

Reply:
I tried that, but liked it better as a value. Then the zero values are automatically present for unit tests. I'll pay the microsecond cost of copying.

Reply:
I see what you mean: why copy all the flags here, if only a subset will be used? Wouldn't it be simpler just to copy a subset in the first place?

It's a little more complicated than that. There's other code that's not part of this PR that is also relevant.

Currently, we have two places in the "action" functions that use flags. The "include" action uses the "dest" flag, and this "print" action uses the "dest", "source", and "spec" flags. Now that we have two cases of accessing flags from an action function, I'm thinking about a clearn future-proof solution rather than having a union-of-subsets-of-flags-that-are-currently-needed-by-action-functions in the stepParams. By placing all the flags here and just pulling the subset that are needed in each action, it's clean and future proof. I'm anticipating needing more flags in our future action functions.